libcontainer/rootfs_linux.go

```diff
@@ -152,14 +153,46 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		}
 		return nil
 	case "tmpfs":
+		copyUp := m.Extensions&configs.EXT_COPYUP == configs.EXT_COPYUP
+		tmpDir := ""
```

Comment:
@cyphar Maybe I am missing a trick. Could you paste a snippet to show what you mean? I need to access tmpDir in both the `copyUp` blocks below now.